### PR TITLE
Add short description to all requests - seeds

### PR DESCRIPTION
**Pull request by GayatriArora**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -278,46 +278,61 @@
...
+Request.create!(user_id: user_seeker_2.id, category_id: category_laptop.id, description: "Saloo needs a laptop to study Social Work.", status: "Completed")
+Request.create!(user_id: user_seeker_5.id, category_id: category_mobile.id, description: "Noor needs a mobile phone to start her own catering business.")
+Request.create!(user_id: user_seeker_7.id, category_id: category_printer.id, description: "Sulaiman needs a desktop computer and a printer to continue his job as an engineer")
+Request.create!(user_id: user_seeker_3.id, category_id: category_desktop.id, description: "Bob needs a desktop and monitor to continue his journey to be a software engineer.")
+Request.create!(user_id: user_seeker_3.id, category_id: category_monitor.id, description: "Bob needs a desktop and monitor to continue his journey to be a software engineer.")
+Request.create!(user_id: user_seeker_4.id, category_id: category_camera.id, description: "Abebe needs a camera so that he can continue his career as a photographer.")
+Request.create!(user_id: user_seeker_8.id, category_id: category_laptop.id, description: "Wafaa needs a laptop to finish her final year of university and find a job.")
```
Comment (RyanofWoods) [inline db/seeds.rb:288]:
```suggestion
Request.create!(user_id: user_seeker_8.id, category_id: category_mobile.id, description: "Wafaa needs a laptop to finish her final year of university and find a job.")
```

File: db/seeds.rb [hunk 2]
```diff
@@ -278,46 +278,61 @@
...
+                user_id: user_seeker_1.id, 
                 product_id: product_macbook.id, 
-                status: "Accepted"
+                status: "Declined"
                 )
-Booking.create!(charity_id: charity2.id, 
-                user_id: user_seeker_5.id, 
+Booking.create!(charity_id: charity1.id, 
+                user_id: user_seeker_8.id, 
                 product_id: product_mobile.id,
```
Comment (RyanofWoods) [inline db/seeds.rb]: I think this needs to be a laptop as this seeker requests a laptop

File: db/seeds.rb [hunk 3]
```diff
@@ -278,46 +278,61 @@
...
+Booking.create!(charity_id: charity1.id, 
+                user_id: user_seeker_8.id, 
                 product_id: product_mobile.id, 
                 status: "Pending"
                 )
-Booking.create!(charity_id: charity2.id, 
-                user_id: user_seeker_3.id, 
+Booking.create!(charity_id: charity4.id, 
+                user_id: user_seeker_5.id, 
                 product_id: product_desktop.id,
```
Comment (RyanofWoods) [inline db/seeds.rb]: I think this needs to be a mobile as the seeker requests a mobile

File: db/seeds.rb [hunk 4]
```diff
@@ -278,46 +278,61 @@
...
                 )
-Booking.create!(charity_id: charity2.id, 
+Booking.create!(charity_id: charity4.id, 
                 user_id: user_seeker_6.id, 
                 product_id: product_ipad.id, 
                 status: "Accepted"
                 )
+Booking.create!(charity_id: charity5.id, 
+                user_id: user_seeker_7.id, 
+                product_id: product_dell_laptop.id,
```
Comment (RyanofWoods) [inline db/seeds.rb]: I think this needs to be a desktop or printer as they requested that

File: db/seeds.rb (hunk 2 above)
Comment (GayatriArora) [reply db/seeds.rb]: Thanks @RyanofWoods , missed that

File: db/seeds.rb [hunk 5]
```diff
@@ -278,46 +278,61 @@
...
                 status: "Accepted"
                 )
+Booking.create!(charity_id: charity5.id, 
+                user_id: user_seeker_7.id, 
+                product_id: product_dell_laptop.id, 
+                status: "Pending"
+                )
+Booking.create!(charity_id: charity3.id, 
+                user_id: user_seeker_3.id, 
+                product_id: product_dell_laptop.id,
```
Comment (RyanofWoods) [inline db/seeds.rb]: I think this needs to be a desktop as the seeker requests this

File: db/seeds.rb [hunk 6]
```diff
@@ -278,46 +278,61 @@
...
+                status: "Pending"
+                )
+Booking.create!(charity_id: charity3.id, 
+                user_id: user_seeker_3.id, 
+                product_id: product_dell_laptop.id, 
+                status: "Pending"
+                )
+Booking.create!(charity_id: charity2.id, 
+                user_id: user_seeker_4.id, 
+                product_id: product_dell_laptop.id,
```
Comment (RyanofWoods) [inline db/seeds.rb]: I think this needs to be a camera as the seeker requests a camera

File: db/seeds.rb [hunk 7]
```diff
@@ -278,46 +278,61 @@
...
-Request.create!(user_id: user_seeker_8.id, category_id: category_laptop.id)
-Request.create!(user_id: user_seeker_6.id, category_id: category_tablet.id, status: "Completed")
+Request.create!(user_id: user_seeker_1.id, category_id: category_laptop.id, description: "Amal needs a laptop to complete her university degree.")
+Request.create!(user_id: user_seeker_2.id, category_id: category_laptop.id, description: "Saloo needs a laptop to study Social Work.", status: "Completed")
+Request.create!(user_id: user_seeker_5.id, category_id: category_mobile.id, description: "Noor needs a mobile phone to start her own catering business.")
+Request.create!(user_id: user_seeker_7.id, category_id: category_printer.id, description: "Sulaiman needs a desktop computer and a printer to continue his job as an engineer")
+Request.create!(user_id: user_seeker_3.id, category_id: category_desktop.id, description: "Bob needs a desktop and monitor to continue his journey to be a software engineer.")
+Request.create!(user_id: user_seeker_3.id, category_id: category_monitor.id, description: "Bob needs a desktop and monitor to continue his journey to be a software engineer.")
```
Comment (RyanofWoods) [inline db/seeds.rb]: Can we make it so descriptions only refer to the request category, i.e. not have multiple categories in the description.
(1 request is for 1 category)

File: db/seeds.rb [hunk 8]
```diff
@@ -88,7 +88,7 @@
                         last_name: "Mir",
                         charity_id: charity2.id,
                         kind: "Seeker",
-                        description: "Saloo is from Bhutan and was studying in university before he had to seek asylum in Britain.He needs a laptop to study social work so that he can work with his local council to support other young refugees .",
+                        description: "Saloo is from Bhutan and was studying in university before he had to seek asylum in Britain. He needs a laptop to study social work so that he can work with his local council to support other young refugees.",
```
Comment (RyanofWoods) [inline db/seeds.rb]: Maybe we can remove about what tech the seekers need from their user.description, as the requests give more information about what they need

**Review by RyanofWoods — CHANGES_REQUESTED** [review]

**Pull request merged**